leds.ato

```diff
@@ -58,18 +58,17 @@ component _KT_0603B from LED:
     signal cathode ~ pin 1
     signal anode ~ pin 2
 
-component _KT_0603G:
+component _KT_0603G from LED:
     # component 0603 Green 509-620mcd
     footprint = "LED0603-RD"
     mpn = "C12624"
-    designator_prefix = "LED"
     v_f = 2.75V
     i_max = 20mA
     # pins
     signal anode ~ pin 2
```

Comment:
This means we're additionally defining anode and cathode. I have a hunch this might've been done before pins are flipped? I'm unsure